src/pages/yllapito/projekti/[oid]/suunnittelu/index.tsx

```diff
@@ -18,19 +18,20 @@ export default function SuunnitteluWrapper() {
 
 function Suunnittelu({ projekti }: { projekti: ProjektiLisatiedolla }): ReactElement {
   const migroitu = projekti?.vuorovaikutusKierros?.tila == VuorovaikutusKierrosTila.MIGROITU;
-  const lukutila = projektiOnEpaaktiivinen(projekti) || !projekti.nykyinenKayttaja.omaaMuokkausOikeuden;
+  const lukutila =
+    projektiOnEpaaktiivinen(projekti) || !projekti.nykyinenKayttaja.omaaMuokkausOikeuden || projekti.nahtavillaoloVaiheJulkaisu;
 
   if (migroitu) {
     return (
-      <SuunnitteluPageLayoutWrapper>
+      <SuunnitteluPageLayoutWrapper lukutila={!!projekti.vuorovaikutusKierrosJulkaisut?.length}>
```

Comment:
Miksi migratoidun suunnittelman lukutila-boolean perustuu julkaisujen olemassaoloon?

Reply:
Tää liitty jotenkin siihen, että jos on migratoitu projekti suunnitteluvaiheeseen, ja sit on tehty nähtävilläolojulkaisu, mut sit palattu nähtävilläolosta suunnitteluun sillä palaamistoiminnolla... Lukutila tarkoittaa, että ei ole mahdollista luoda uutta suunnitteluvaiheen kutsua. Muuten on.

Tässä pitäis olla oikeasti siis !!projekti.nahtavillaoloVaiheJulkaisu. Muutan sen nyt sillai. Eli jos suunnitteluvaihe on migroitu ja nähtävilläolovaiheessa on tehty julkaisu, ei voida enää luoda uuttaa suunnitteluvaiheen kutsua. Sen sijaan jos suunnitteluvaihe on migroitu, ja nähtävilläolovaiheessa ei ole vielä yhtään julkaisua, tarkoittaa se sitä, että saatetaan olla tilanteessa, jossa ollaan palattu nähtävilläolosta suunnitteluun, jolloin halutaan voida tehdä uusi suunnitteluvaiheen kutsu.

Reply:
Joo nyt hiffaan.